ui/register_react_component.md

```diff
@@ -0,0 +1,98 @@
+### Register react component
+In ManageIq project there is a way of sharing components, getting their state and control their flow by something which is called ManageIq component.
+
+This registry and usage consists of three steps, define blueprint which holds the react component with it's props. Then you will define it inside shared components (use blueprint created in step 1). And when someone decides to use such component he will call function newInstance with custom props and element to which such component is bound to.
```

Comment:
it's = it is
its = possessive 

... in many places :)

Reply:
s/which holds/that holds